### PR TITLE
Sliders documentation

### PR DESCRIPTION
**Pull request by Charles-ErickBG**
### Acceptance Criteria

-   [ ] The proposed changes are covered by unit tests
-   [ ] The potential breaking changes are clearly identified
-   [ ] [README.md](https://github.com/coveo/react-vapor/blob/master/README.md) is adjusted to reflect the proposed changes (if relevant)

### CODE REVIEW

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+### Behavior
+
+-   Adjust the size and behavior of the rail so that users have access to the precision level they need, no more, no less.
+
+-   Always set the lowest value on the left side and the highest on the right.
+
+-   Jumping to evenly spaced marks along the rail may be enabled to help users select discrete values.
+
+-   Discrete values should be selectable by clicking the corresponding value label.
```
Comment (jpmarceau) [inline packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: That reads a bit like the previous line... is there something I'm not seeing? Could the two be combined?

File: packages/react-vapor/src/components/slider/examples/SliderExamples.tsx [hunk 2]
```diff
@@ -2,6 +2,9 @@ import * as React from 'react';
 import {Slider} from '../Slider';
 
 export class SliderExamples extends React.Component<any, any> {
+    static description =
+        'Sliders are used to adjust contiguous values (e.g., screen brightness settings) that do not require a high level of precision or a range of related values (e.g., screen resolution settings).';
```
Comment (jpmarceau) [inline packages/react-vapor/src/components/slider/examples/SliderExamples.tsx]: Do you mean 
"...that do not require (a high level of precision), nor (a range of related values)..."
or "...that do not require (a high level of precision or a range of related values)..." ?

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### General
+
+-   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect), or [checkbox](#/components/Checkbox) instead.
```
Comment (kpoulincoveo) [inline packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: I'm not sure how a slider could be confused with toggles or checkboxs... (which is not a binary settings form anyway). 
It think the general guideline should focus more about why/when we should Slider over a Numeric Input

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md (hunk 1 above)
Comment (Charles-ErickBG) [reply packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: Yes. Combining them in the same bullet point would do it, IMO.

File: packages/react-vapor/src/components/slider/examples/SliderExamples.tsx (hunk 2 above)
Comment (Charles-ErickBG) [reply packages/react-vapor/src/components/slider/examples/SliderExamples.tsx]: the first one on top :)

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+## Guidelines
+
+### General
+
+-   For binary settings, use a [slide toggle](#/styles/form-controls/slide-toggle), [toggle](#/components/FlatSelect) or [checkbox](#/components/Checkbox) instead.
+
+### Labels
```
[the remark this answers is not in the document]
Comment (Charles-ErickBG) [reply packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: @jpmarceau

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md (hunk 3 above)
Comment (kpoulincoveo) [inline packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: I think we should refer to Numeric Input instead. They are closer in terms of intent then checkbox and toggles. 
(and sorry if this is the third time you see this message, seems like it doesn't want to show up)

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (jpmarceau) [reply packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: did that while I was there :)

File: packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md (hunk 3 above)
Comment (Charles-ErickBG) [reply packages/react-vapor/src/components/slider/examples/SliderExamples.1.use.md]: Ok. And how would you choose between Numeric Input and Slider?

**Comment by GermainBergeron** [comment]
@Charles-ErickBG Is this ready to merge? It's been a long time without any activity

**Pull request merged**